eclass/gkrellm-plugin.eclass

```diff
@@ -55,7 +62,7 @@ EXPORT_FUNCTIONS src_install
 
```

Comment:
put the `EXPORT_FUNCTIONS` as the last line of this file

Reply:
Done. Added just before the closing `fi`